.github/workflows/ci.yml

```
@@ -0,0 +1,99 @@
+          go-version-file: 'go.mod'
+          cache: true
+          cache-dependency-path: '**/go.sum'
+
+      - name: Check Go version
+        run: go version
+
+      - name: Install and run Gosec
+        run: |
+          go install github.com/securego/gosec/v2/cmd/gosec@latest
+          gosec -fmt sarif -out gosec-results.sarif ./... || true
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)